srcpkgs/loupe/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'loupe'
+pkgname=loupe
+version=45.3
+revision=1
+build_style=meson
+build_helper=rust
+hostmakedepends="cargo desktop-file-utils gettext gtk4-update-icon-cache
+ itstool pkg-config"
+makedepends="lcms2-devel libadwaita-devel libgweather-devel rust-std"
+depends="glycin-loaders"
+short_desc="Simple image viewer for GNOME"
+maintainer="chrysos349 <chrysostom349@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/GNOME/loupe"
+changelog="https://gitlab.gnome.org/GNOME/loupe/-/raw/main/NEWS"
+distfiles="${GNOME_SITE}/loupe/${version%.*}/loupe-${version}.tar.xz"
+checksum=f65f2d1208d085ab5ffa998dd43c92fe952272d4d59b51cfecc11ebdffca2d86
+
+post_patch() {
+	if [ "$CROSS_BUILD" ]; then
+		vsed -i src/meson.build \
+			-e "s%rust_target /%'${RUST_TARGET}' / rust_target /%"
+	fi
+}
+
+do_install() {
+	DESTDIR=${DESTDIR} meson install -C build --no-rebuild
```

Comment:
Why are you doing this instead of `make_install_args`?

Reply:
Sorry, I didn't realize you are doing meson install --no-rebuild which is what gets called by `ninja install` anyway, not sure the point of this.

Reply:
i removed this section, and git dep from glycin-loaders. i didn't update them yet, because loupe-46.0 requires newer gtk4 (at the same time it ran fine with the current gtk4 from the repo, which means the newer is not strictly required (maybe)). also loupe-45.3 won't work with newer glycin-loaders.